src/DIRAC/DataManagementSystem/Client/ConsistencyInspector.py

```diff
@@ -498,8 +498,7 @@ def set_prod(self, value):
       value = int(value)
       res = self.transClient.getTransformation(value, extraParams=False)
       if not res['OK']:
-        S_ERROR(errno.ENOENT, "Couldn't find transformation %d: %s" %
-                (value, res['Message']))
+        raise Exception("Couldn't find transformation %d: %s" % (value, res['Message']))
```

Comment:
Shouldn't the S_ERROR be just returned ?

Reply:
It can't be as it's used in a `property`'s setter so there is no way to access the return value.